source/cli-translate.js

```diff
@@ -0,0 +1,128 @@
+var imagemagick_cli_lists = null;
+
+function addSelectOptions(node)
+{
+    var selectCliLanguageNode = document.createElement("select");
+    selectCliLanguageNode.className += "btn text-dark bg-light mx-4";
+    selectCliLanguageNode.style.fontSize=".75rem";
+    selectCliLanguageNode.style.float="right";
+    selectCliLanguageNode.style.clear="both";
+    selectCliLanguageNode.style.position="relative";
+    selectCliLanguageNode.style.top="-18px";
+    selectCliLanguageNode.style.height="22px";
+    selectCliLanguageNode.style.padding="0px";
+    selectCliLanguageNode.onchange = function(){setCliChanged(this)};
+    selectCliLanguageNode.title = "Change Operating System Syntax";
+    var selectCliLanguageOptions = ["Linux","Windows","Bat File"];
+
+    for(let option of selectCliLanguageOptions){
+        var optionNode = document.createElement("option");
+        optionNode.text=option;
+        selectCliLanguageNode.appendChild(optionNode);
+    }
+    node.after(selectCliLanguageNode);
+    return selectCliLanguageNode;
+}
+
+const imagemagick_linux_to_windows_replacements = new Map([
```

Comment:
There are two separate maps of replacements. One for Windows command line, the other for additional windows bat files.

Reply:
Sounds like a good plan to do this.